capstone/src/main/java/com/google/univiz/SuggestionAPIData.java

```diff
@@ -0,0 +1,15 @@
+package com.google.univiz;
+
+import java.util.List;
+
+/**
+ * Communicates directly with the REST API to get a list of college suggestions related to a client
+ * query
+ */
+public interface SuggestionDataAPI {
+  /**
+   * Takes a list of college names and returns a list of SuggestionData objects, where each object
+   * is unique to each college name providede in the first list
+   */
+  List<SuggestionData> getCollegeSuggestions(List<String> collegeNames);
```

Comment:
Chris pointed this out earlier, but are we expecting a list of college names from the client? Does it make sense to expect this?

Reply:
fixed this in the other PR, will update it here :)